packages/common/src/GRPCTypeTranslation.ts

```diff
@@ -1652,6 +1652,50 @@ export function instanceStateKVPair(
     };
 }
 
+export function ipInfo(ip: v2.IpInfo): v1.IpInfo {
+    return {
+        ipIdentity: unwrap(ip.identity?.value),
+        ipDescription: unwrap(ip.description),
+        ipVerifyKey: unwrapValToHex(ip.verifyKey),
+        ipCdiVerifyKey: unwrapValToHex(ip.cdiVerifyKey),
+    };
+}
+
+export function arInfo(ar: v2.ArInfo): v1.ArInfo {
+    return {
+        arIdentity: unwrap(ar.identity?.value),
+        arDescription: unwrap(ar.description),
+        arPublicKey: unwrapValToHex(ar.publicKey),
+    };
+}
```

Comment:
I feel like I've already implemented this once? :thinking: 
Maybe that can be merged with this.

Reply:
You're right, a helper function can be implemented.